SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/LegacySpatialWorkerConnection.cpp

```diff
@@ -69,100 +70,106 @@ void USpatialWorkerConnection::DestroyConnection()
 	KeepRunning.AtomicSet(true);
 }
 
-TArray<Worker_OpList*> USpatialWorkerConnection::GetOpList()
+TArray<OpList> ULegacySpatialWorkerConnection::GetOpList()
 {
-	TArray<Worker_OpList*> OpLists;
+	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
+	if (SpatialGDKSettings->bRunSpatialWorkerConnectionOnGameThread)
+	{
+		QueueLatestOpList();
+	}
+
+	TArray<OpList> OpLists;
 	while (!OpListQueue.IsEmpty())
 	{
-		Worker_OpList* OutOpList;
+		OpList OutOpList;
 		OpListQueue.Dequeue(OutOpList);
-		OpLists.Add(OutOpList);
+		OpLists.Add(MoveTemp(OutOpList));
```

Comment:
I think this may be reversing the order of the op lists

Reply:
Ah I see its in legacy though, maybe it no longer matters :D

Reply:
Where is it reversing them? I think this is the same order as it's always been.

Reply:
Ah my bad I thought TQueue tail enqueue'd